### DISCUSSION TITLE
Consider adding Oppaitime indexer

### DISCUSSION THREAD

**Discussion by eoop** [autobrr/autobrr#330] [Ideas]
Since indexers like Empornium are supported, I'd like to suggest adding support for Oppaitime. It's an indexer for Japanese focused porn.

**Comment by zze0s** [comment]
Hi! Yeah that's doable but I need you to gather all the info. We currently only support indexers with IRC announces.

# Indexer request

-  Do they have irc announce? Yes / No
-  Link to existing autodl-irssi .trackers file: Check wiki, forums and autodl-trackers repo

## Announce examples

Please provide 1-5 different announces. And change release titles for good meassure

- `SOME LONG ANNOUNCE`
- `OTHER ANNOUNCE`

## IRC Network

- Network name: TrackerNetwork
- Network address: irc.network.tld
- Port: 6697 is common for TLS
- SSL/TLS: Yes / No
- NickServ registration required? Check wiki, guides etc: Yes / No
- Is there a required format for the nick? Like `username|bot`? Check wiki, guides etc
- Invite command: Eg. `invitebot announce USERNAME IRCKEY`

## Bonus

Check their categories in the announce an on site

Announce category - Site category name

**Reply by zze0s** [reply]
@eoop no sure if you saw this so I'm pinging you now :)

**Comment by eoop** [comment]
IRC Announce: Yes
[Link to autodl-trackers file](https://github.com/autodl-community/autodl-trackers/blob/master/trackers/Oppaitime.tracker)

Announces:
https://t1.oppaiti.me/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/announce
https://t2.oppaiti.me/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/announce
https://t3.oppaiti.me/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/announce
https://t4.oppaiti.me/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx/announce

### IRC Network:
Network name:
Network address: irc.oppaiti.me
Port: 6697 (SSL), 6667 (Non-SSL)
NickServ: Does not appear to be required
Nick/Identification:

> To identify with Udon:
/MSG Udon KNOCK [channel] [username] [irckey]
Replacing [channel] with the IRC channel you wish to join, [username] with your username on the site, and [irckey] with the key you previously set up.
>
>If your key is correct, Udon will join you to the channel you requested. You will need to KNOCK again on any channels you wish to join or rejoin.

Bots:
>You are permitted to have bots on IRC, but only in the following channels:
>#oppaitime-announce
>#oppaitime-requests

### Bonus:
Categories on site:
- Movies
- Anime
- Manga
- Games
- Audio
- Other
